docs/vectorizer/api-reference.md

```diff
@@ -128,7 +164,8 @@ in other management functions.
 | Name             | Type                                                   | Default                           | Required | Description                                                                                        |
 |------------------|--------------------------------------------------------|-----------------------------------|----------|----------------------------------------------------------------------------------------------------|
 | source           | regclass                                               | -                                 | ✔        | The source table that embeddings are generated for.                                                |
-| destination      | name                                                   | -                                 | ✖        | Set the name of the table embeddings are stored in, and the view with both the original data and the embeddings.<br>The view is named `<destination>`, the embedding table is named `<destination>_store`.<br>You set destination to avoid naming conflicts when you configure additional vectorizers for a source table.                              |
+| name             | text                                                   | Auto-generated                    | ✖        | A unique name for the vectorizer. If not provided, it's auto-generated based on the destination type:<br>- For table destination: `[target_schema]_[target_table]`<br>- For column destination: `[source_schema]_[source_table]_[embedding_column]`<br>Must follow snake_case pattern `^[a-z][a-z_0-9]*$`                              |
```

Comment:
don't we need to add the `if_not_exists` parameter here as well?

Reply:
yeh I forgot, added!